nodejs/v8-community/cache/booster.yaml

```diff
@@ -0,0 +1,19 @@
+metadata:
+  runsOn:
+    - '!starter'
+    - '!osio'
+name: Node.js Cache Booster
+description: Demonstrate how to use a cache server from a Node.js application
+source:
+  git:
+    url: https://github.com/bucharest-gold/nodejs-cache
+    ref: master
+environment:
+  staging:
+    source:
+      git:
+        ref: v1.0.1
```

Comment:
Should these version numbers be 2.0?

Reply:
not for the v8 boosters.  i kept those as 1.x.x